tests/unit/s2n_security_policies_test.c

```diff
@@ -153,7 +153,7 @@ int main(int argc, char **argv)
                 int max = security_policy->signature_preferences->signature_schemes[i]->maximum_protocol_version;
                 s2n_signature_algorithm sig_alg = security_policy->signature_preferences->signature_schemes[i]->sig_alg;
 
-                if (min == S2N_TLS13 || max >= S2N_TLS13) {
+                if (min <= S2N_TLS13 || max >= S2N_TLS13) {
```

Comment:
What if the min is S2N_TLS10 and the max is S2N_TLS11? I don't think `has_tls_13_sig_alg` should be true in this case. Would something like this be more accurate maybe?
```suggestion
                if (min <= S2N_TLS13 && (max == 0 || max >= S2N_TS13)) {
```

Reply:
Oh nice catch, I missed how sketchy that check already was lol. I cleaned it up a bit.